src/applications/personalization/profile/components/account-security/AccountSecurityTables.jsx

```diff
@@ -5,117 +5,98 @@ import { mfa } from '~/platform/user/authentication/utilities';
-          </List.ContentIncomplete>
-        </List.Item>
-
-        <List.Item complete={isMultifactorEnabled}>
-          <List.HeadingComplete>
+    <List>
+      <List.Item complete={isIdentityVerified}>
+        <List.HeadingComplete>Verify your identity</List.HeadingComplete>
+
+        <List.ContentComplete>
+          We’ve verified your identity.
+        </List.ContentComplete>
+
+        <List.HeadingIncomplete>
+          <a href="/verify">Verify your identity</a>
+        </List.HeadingIncomplete>
+
+        <List.ContentIncomplete>
+          Verify your identity to view your complete profile.
+        </List.ContentIncomplete>
+      </List.Item>
+
+      <List.Item complete={isMultifactorEnabled}>
+        <List.HeadingComplete>Add 2-factor authentication</List.HeadingComplete>
+
+        <List.ContentComplete>
+          You’ve added an extra layer of security to your account with 2-factor
+          authentication.
+        </List.ContentComplete>
+
+        <List.HeadingIncomplete>
+          <button
+            onClick={mfaHandler}
+            className="va-button-link vads-u-font-family--serif vads-u-font-weight--bold vads-u-font-size--h4"
+            type="button"
+          >
             Add 2-factor authentication
-          </List.HeadingComplete>
-
-          <List.ContentComplete>
-            You’ve added an extra layer of security to your account with
-            2-factor authentication.
-          </List.ContentComplete>
-
-          <List.HeadingIncomplete>
-            <button
-              onClick={mfaHandler}
-              className="va-button-link vads-u-font-family--serif vads-u-font-weight--bold vads-u-font-size--h4"
-              type="button"
-            >
-              Add 2-factor authentication
-            </button>
-          </List.HeadingIncomplete>
-
-          <List.ContentIncomplete>
-            Add an extra layer of security (called 2-factor authentication).
-            This helps to make sure only you can access your account—even if
-            someone gets your password.
-          </List.ContentIncomplete>
-        </List.Item>
-      </List>
-    </section>
+          </button>
+        </List.HeadingIncomplete>
+
+        <List.ContentIncomplete>
+          Add an extra layer of security (called 2-factor authentication). This
+          helps to make sure only you can access your account—even if someone
+          gets your password.
+        </List.ContentIncomplete>
+      </List.Item>
+    </List>
   );
 };
 
-AccountSetupTable.propTypes = {
+AccountSetupList.propTypes = {
   isIdentityVerified: PropTypes.bool.isRequired,
   isMultifactorEnabled: PropTypes.bool.isRequired,
-  title: PropTypes.string.isRequired,
 };
 
 export const AccountSecurityTables = ({
   signInServiceName,
   isIdentityVerified,
   isMultifactorEnabled,
 }) => {
+  const data = (
+    <AccountSetupList
+      {...{
+        isIdentityVerified,
+        isMultifactorEnabled,
+      }}
+    />
+  );
```

Comment:
Note: I'm not crazy about declaring this as a variable instead of just passing the component directly to the `data` prop of the second ProfileInfoCard, but for whatever reason eslint and prettier were causing conflicting formatting when I tried to inline the multiline component as a prop. 🤷🏼